### PR TITLE
docstyle improved for Bio.Pathway, Bio.NeuralNetwork and Bio.Blast

### PR DESCRIPTION
**Pull request by Gasta88**
This pull request addresses issue #1473

### CODE REVIEW

File: Bio/Blast/Applications.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def __init__(self, cmd=None, **kwargs):
         AbstractCommandline.__init__(self, cmd, **kwargs)
 
     def _validate_incompatibilities(self, incompatibles):
-        """Used by the BLAST+ _validate method (PRIVATE)."""
+        """Use _validate method by the BLAST+ (PRIVATE)."""
```
Comment (peterjc) [inline Bio/Blast/Applications.py]: The meaning has changed, how about:

```python
"""Validate parameters for incompatibilities (PRIVATE).

Used by the _validate method.
"""
```

File: Bio/Blast/NCBIXML.py [hunk 2]
```diff
@@ -252,53 +252,53 @@ def _end_BlastOutput_version(self):
                 self._header.date = parts[2]
 
     def _end_BlastOutput_reference(self):
-        """A reference to the article describing the algorithm (PRIVATE).
+        """Yield the article describing the algorithm (PRIVATE).
```
Comment (peterjc) [inline Bio/Blast/NCBIXML.py]: Yield implies a generator/iterator. Return seems wrong here - the function as no return value (Python default is just ``None``). How about:

```python
"""Record any article reference describing the algorithm (PRIVATE)."""
```

File: Bio/Blast/NCBIXML.py [hunk 3]
```diff
@@ -252,53 +252,53 @@ def _end_BlastOutput_version(self):
...
-        """The database(s) searched (PRIVATE).
+        """Yield the database(s) searched (PRIVATE).
 
         Save this to put on each blast record object
         """
         self._header.database = self._value
 
     def _end_BlastOutput_query_ID(self):
-        """The identifier of the query (PRIVATE).
+        """Yield the identifier of the query (PRIVATE).
```
Comment (peterjc) [inline Bio/Blast/NCBIXML.py]: Similar comment to above, how about  ``Record the identifier of the query (PRIVATE).`` instead?

File: Bio/Blast/NCBIXML.py [hunk 4]
```diff
@@ -252,53 +252,53 @@ def _end_BlastOutput_version(self):
...
 
         Important in old pre 2.2.14 BLAST, for recent versions
         <Iteration_query-ID> is enough
         """
         self._header.query_id = self._value
 
     def _end_BlastOutput_query_def(self):
-        """The definition line of the query (PRIVATE).
+        """Yield the definition line of the query (PRIVATE).
```
Comment (peterjc) [inline Bio/Blast/NCBIXML.py]: Same problem as above

File: Bio/Blast/NCBIXML.py [hunk 5]
```diff
@@ -252,53 +252,53 @@ def _end_BlastOutput_version(self):
...
 
         Important in old pre 2.2.14 BLAST, for recent versions
         <Iteration_query-def> is enough
         """
         self._header.query = self._value
 
     def _end_BlastOutput_query_len(self):
-        """The length of the query (PRIVATE).
+        """Yield the length of the query (PRIVATE).
```
Comment (peterjc) [inline Bio/Blast/NCBIXML.py]: Same as above, and applies to all the other private ``_end...`` methods which are used by the parse to capture/record information from the XML into our data structure.

File: Bio/Blast/NCBIXML.py [hunk 6]
```diff
@@ -252,53 +252,53 @@ def _end_BlastOutput_version(self):
...
-        """A reference to the article describing the algorithm (PRIVATE).
+        """Record any article reference describing the algorithm (PRIVATE).
 
         Save this to put on each blast record object
         """
         self._header.reference = self._value
 
     def _end_BlastOutput_db(self):
-        """The database(s) searched (PRIVATE).
+        """Yield the database(s) searched (PRIVATE).
```
Comment (peterjc) [inline Bio/Blast/NCBIXML.py]: Yield --> record

File: Bio/Blast/NCBIXML.py [hunk 7]
```diff
@@ -355,7 +355,7 @@ def _end_Parameters_gap_extend(self):
                                          int(self._value))
 
     def _end_Parameters_filter(self):
-        """Filtering options (-F) (PRIVATE)."""
+        """Record a filtering options (-F) (PRIVATE)."""
```
Comment (peterjc) [inline Bio/Blast/NCBIXML.py]: Options is plural, so "Record a filtering options" --> "Record filtering options"?

Comment (Gasta88) [reply Bio/Blast/NCBIXML.py]: Well, this was pretty dumb xD

Comment (peterjc) [reply Bio/Blast/NCBIXML.py]: Not really. You've been editing so many of these that little mistake can easily happen.

**Comment by peterjc** [comment]
Thank you!

**Pull request merged**